pkg/handler/repositories.go

```diff
@@ -516,6 +517,35 @@ func (rh *RepositoryHandler) introspect(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+// Update godoc
+// @Summary      Get the GPG key file for a repository
+// @ID           getGpgKeyFile
+// @Description  Get the GPG key file for a repository.
+// @Tags         repositories
+// @Accept       json
+// @Produce      text/plain
+// @Param  uuid       path    string  true  "Repository ID."
+// @Success      200 {object}  api.RepositoryResponse
```

Comment:
api.RepositoryResponse isn't quite right.  Should it just be 'string' ?

Reply:
yep!

Reply:
Works now, thank you